### PR TITLE
API Route: add money

### PR DESCRIPTION
**Pull request by malivinayak**

### CODE REVIEW

**Comment by malivinayak** [comment]
@OutdatedGuy API tested with UI

File: src/backend/routes/wallet/addMoney.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    oracledb.initOracleClient({ libDir: libPath });
+}
+
+const addMoney = async (req, res) => {
+
+    try {
+        const { token } = req.body;
+        let { amount } = req.body;
+        amount = parseInt(amount)
+        if (amount < 10) {
```
Comment (OutdatedGuy) [inline src/backend/routes/wallet/addMoney.js]: Do 100 instead of 10. Looks standard.

File: src/backend/routes/wallet/addMoney.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+            options = {
+                outFormat: oracledb.OUT_FORMAT_OBJECT,
+            };
+
+            const getBalQuery = `select WALLET_BALANCE from TRIP_MANAGEMENT_SYSTEM.client where TOKEN = :1`;
+            const getCurrentBalance = await connection.execute(getBalQuery, [token], options);
+            if (getCurrentBalance.rows[0] === undefined) {
+                return res.status(403).send({
+                    message: "Invalid Token",
```
Comment (OutdatedGuy) [inline src/backend/routes/wallet/addMoney.js]: Don't send messages like "invalid token" to frontend. The users are not supposed to know this kind of failures. Instead say "Something went wrong, please refresh the page and retry".

File: src/backend/routes/wallet/addMoney.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                    code: 400,
+                });
+            }
+            let currentBalance = getCurrentBalance.rows[0].WALLET_BALANCE;
+            currentBalance = currentBalance + amount;
+
+            query = `update TRIP_MANAGEMENT_SYSTEM.client SET WALLET_BALANCE = ${currentBalance} where TOKEN = '${token}'`;
+            await connection.execute(query, [], { autoCommit: true });
+            return res.send({
+                message: `🎉Balance Deposited success.......
```
Comment (OutdatedGuy) [inline src/backend/routes/wallet/addMoney.js]: Add space after emoji.

File: src/backend/routes/wallet/addMoney.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+To Check updated balance click "Get Wallet Balance" button again`,
+                status: "success",
+                code: 200,
+            });
+
+        } catch (err) {
+            console.log(" Error at Data Base : " + err);
+            return res.status(500).send({
+                message: "Balance Deposit Failed!!!",
```
Comment (OutdatedGuy) [inline src/backend/routes/wallet/addMoney.js]: Also add "Please try again later" message

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]

**Review by OutdatedGuy — APPROVED** [review]

**Pull request merged**